bin/run-tests-in-docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+# Synopsis:
+# Test the test runner Docker image by running it against a predefined set of 
+# solutions with an expected output.
+# The test runner Docker image is built automatically.
+
+# Output:
+# Outputs the diff of the expected test results against the actual test results
+# generated by the test runner Docker image.
+
+# Example:
+# ./bin/run-tests-in-docker.sh
+
+# Stop executing when a command returns a non-zero return code
+set -e
+
+# Build the Docker image
+docker build --rm -t exercism/python-test-runner .
+
+# Run the Docker image using the settings mimicking the production environment
+docker run \
+    --rm \
+    --network none \
+    --read-only \
+    --mount type=bind,src="${PWD}/test",dst=/opt/test-runner/test \
+    --mount type=volume,dst=/tmp \
+    --workdir /opt/test-runner \
+    --entrypoint pytest \
+    exercism/python-test-runner
```

Comment:
```suggestion
    exercism/python-test-runner -vv
```

Reply:
This will set the pytest output to verbose, which will print out the name of each test and a PASS/FAIL, as well as a full diff for any failures.  It is a LOT of diff, but can be really helpful for troubleshooting where things might have gone wrong.  If it becomes a problem, I am assuming we can set the delete on the logs to a short timespan, or remove the logs on review.

**Fun facts:**  
- Changing Python versions will often break certain tests, due to Python printing out its version number when an error is thrown.
- Changing OS versions often breaks certain tests because there are subtle changes in how line breaks and spaces happen...among other things.
- Now added to the list -- using `-vv` in the `pytest.ini` file under `adopts` will also break 21 of these tests, due to its interaction with the different `style` flags being tested.

Reply:
That makes total sense. I just hadn't spotted it when copying. My bad!